samples/cors/arm-template.json

```diff
@@ -0,0 +1,159 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "baseName": {
+      "type": "string",
+      "defaultValue": "[resourceGroup().name]",
+      "metadata": {
+        "description": "The base resource name."
+      }
+    }
+  },
+  "variables": {
+    "serviceName": "[concat(parameters('baseName'), '-api')]",
+    "vaultName": "[concat(parameters('baseName'), '-keyvault')]"
+  },
+  "resources": [
+    {
+      "type": "Microsoft.ApiManagement/service",
+      "apiVersion": "2019-01-01",
+      "name": "[variables('serviceName')]",
+      "location": "[resourceGroup().location]",
+      "properties": {
+        "notificationSenderEmail": "apimgmt-noreply@mail.windowsazure.com",
```

Comment:
Better to leave blank? Or is this just the default?

Reply:
Yes, better to leave blank 😄 good catch